package/batocera/core/batocera-configgen/configgen/configgen/generators/xash3d_fwgs/xash3dFwgsGenerator.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import glob
+import os
+
+import Command
+from generators.Generator import Generator
+import controllersConfig
+
+
+class Xash3dFwgsGenerator(Generator):
+
+    def generate(self, system, rom, playersControllers, gameResolution):
+        # Useful options for debugging:
+        # -log        # Log to /userdata/roms/xash3d_fwgs/engine.log
+        # -dev 5      # Verbose logging
```

Comment:
Maximum log level is 2.
https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/host.c#L81
https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/host.c#L826
https://github.com/FWGS/xash3d-fwgs/blob/master/engine/common/common.h#L90

Reply:
Thanks, fixed
I took `-dev 5` from this guide btw https://github.com/FWGS/xash3d-fwgs/tree/master/contrib/mittorn